openreview/conference/templates/reviewerWebfield.js

```diff
@@ -378,21 +378,3 @@ $('#group-container').on('click', 'a.note-contents-toggle', function(e) {
   var updated = $(this).text() === hiddenText ? visibleText : hiddenText;
   $(this).text(updated);
 });
-
-$('#group-container').on('click', 'a.send-reminder-link', function(e) {
```

Comment:
Why was this handler removed? Is it no longer being used?

Reply:
do we send reminders from the reviewers console?

I think this was a copy/paste from the AC console, correct me if I'm wrong.

Reply:
I took at look and I think you are right.